src/app/features/gossip/select-gossip/select-gossip.service.ts

```diff
@@ -10,4 +10,32 @@ export class SelectGossipService extends SelectService<GossipMenu> {
   constructor(readonly queryService: MysqlQueryService, public handlerService: GossipHandlerService) {
     super(queryService, handlerService, GOSSIP_MENU_TABLE, GOSSIP_MENU_ID, null, GOSSIP_MENU_SEARCH_FIELDS);
   }
+
+  onSearch(): void {
```

Comment:
more importantly, why overriding this method instead of just editing `GOSSIP_MENU_SEARCH_FIELDS` in `gossip-menu.type.ts`